rpcs3/Emu/Cell/PPUModule.cpp

```diff
@@ -1161,7 +1161,7 @@ static void ppu_check_patch_spu_images(const ppu_module& mod, const ppu_segment&
 
 					// FIXME: This seems to terminate SPU code prematurely in some cases
 					// Likely due to absolute branches
-					if (spu_thread::is_exec_code(instruction, {reinterpret_cast<const u8*>(ls_segment.data()), ls_segment.size()}, 0))
+					if (spu_thread::is_exec_code(::narrow<u32>(instruction), {reinterpret_cast<const u8*>(ls_segment.data()), ls_segment.size()}, 0))
```

Comment:
make it u32 instruction; instead with static cast

Reply:
like this? idk if the find is guaranteed to be in scope of u32

Reply:
yeah it is 👍

Reply:
actually, won't this break the umax check?

Reply:
Suprisingly no

Reply:
Ah, because casting from 0xFFFFFFFFFFFFFFFF to u32 is 0xFFFFFFFF.

Reply:
Anything else?